src/ci/scripts/install-mingw.sh

```diff
@@ -61,12 +43,5 @@ if isWindows; then
             mingw-w64-$arch-gcc \
             mingw-w64-$arch-python # the python package is actually for python3
         ciCommandAddPath "$(ciCheckoutPath)/msys2/mingw${bits}/bin"
-    else
-        mingw_dir="mingw${bits}"
-
-        curl -o mingw.7z "${MIRRORS_BASE}/${mingw_archive}"
-        7z x -y mingw.7z > /dev/null
```

Comment:
Don't we need to keep this code, apart from the gdb replacement? Otherwise we're not actually performing the download anywhere?

Reply:
Honestly I have no clue what happened there but you are right.